deploy_via_trigger.sh

```diff
@@ -21,21 +21,29 @@ handle_error() {
 }
 trap 'handle_error' ERR
 
-while getopts p:r:t: flag
+while getopts p:t: flag
 do
     case "${flag}" in
         p) PROJECT_ID=${OPTARG};;
-        r) REGION=${OPTARG};;
         t) IMAGE_TAG=${OPTARG};;
     esac
 done
 
+# Iterate over the infra manager location to identify the deployment
+# currently one deployment per project is only supported
+# in future if multiple deployments are supported per project this will need to change
+for i in "us-central1" "europe-west1" "asia-east1"
```

Comment:
nit: Can we store the region in a variable, let's say: IM_SUPPORTED_REGIONS and use that in for loop? 

```
IM_SUPPORTED_REGIONS=("us-central1" "europe-west1" "asia-east1")

for REGION in "${IM_SUPPORTED_REGIONS[@]}"; do
    DEPLOYMENT_NAME=$(gcloud infra-manager deployments list --location "${REGION}" \
                        --filter="labels.goog-solutions-console-deployment-name:* AND \
                        labels.goog-solutions-console-solution-id:three-tier-web-app" \
                        --format='value(name)' )
    if [ -n "$DEPLOYMENT_NAME" ]; then
        break
    fi
done
```